.storybook/themes/news.scss

```
@@ -822,6 +822,15 @@
 				video-player-inline-video: (
 					gap: 0,
 				),
+				links-bar: (
+					display: flex,
+					justify-content: center,
+					align-items: center,
+					gap: var(--global-spacing-2),
+				),
+				links-bar__menu-link: (
```

```suggestion
				links-bar-menu-link: (
```

For the token names, we've been using dashes to separate the names. I'm sorry if this isn't clear. Do you remember talking about the difference between tokens and scss classes? They don't have to be the same. The token names, by our convention, are separated by dashes and the scss classes use bem naming